src/uuid/rng/init.lua

```diff
@@ -78,7 +78,42 @@ function rng.luasystem()
   return sys.random
 end
 
+----------------------------------------------------------------------------
+-- Returns an rng that implements Windows' RtlGenRandom function which is
+-- a good source of randomness on all modern versions of Windows.
+-- Requires LuaJIT or a LuaJIT-compatible FFI library.
+-- @treturn function A function that returns `n` random bytes, signature: `byte_string, err = func(n)`
+-- @usage
+-- local uuid = require "uuid"
+-- uuid.set_rng(uuid.rng.win_ffi())
+function rng.win_ffi()
+  local ok, ffi = pcall(require, "ffi")
+  if not ok then 
+      ok, ffi = pcall(require, "cffi")
```

Comment:
which ffi module is this, can we add it to the docs?

Reply:
"ffi" is luajit's default ffi module, "cffi" is [cffi-lua](https://github.com/q66/cffi-lua), a LuaJIT-compatible ffi library.

Reply:
yes, a link would be nice, the LuaJIT one is well known, but the others less so